Source/Diagnostics/WarpXOpenPMD.cpp

```diff
@@ -1216,42 +1260,87 @@ WarpXOpenPMDPlot::WriteOpenPMDFieldsAll ( //const std::string& filename,
 
```

Comment:
writeOpenPMDFieldsAll (...) in RZ only works on thetaMode files, i.e. on data files initialized in FullDiagnostics::AddRZModesToDiags.  This made sense to me, but does require some care in usage and could be relaxed